flips/yyyymmdd-flip-template.md

```diff
@@ -0,0 +1,78 @@
+# Title of FLIP
+
+| Status        | (Proposed / Accepted / Implemented / Obsolete)       |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #**    | [NNN](https://github.com/onflow/flow/pull/NNN) (update when you have PR #)|
+| **Author(s)** | My Name (me@example.org), AN Other (you@example.org) |
+| **Sponsor**   | A N Expert (core-contributor@example.org)            |
+| **Updated**   | YYYY-MM-DD                                           |
+| **Obsoletes** | FLIP it replaces, else remove this header            |
+
```

Comment:
As noted above, I think it might be a good idea to add a status.

Inspiration could be again https://github.com/apple/swift-evolution/blob/master/process.md#proposal-states

Would be good to have e.g.

* **Rejected**: The proposal has been reviewed and been rejected. [Link to reasoning]
* **Accepted**: The proposal has been accepted and is either awaiting implementation or is actively being implemented.
* **Implemented (in VERSION)**: The proposal has been implemented.